Bigtable/src/Admin/V2/resources/bigtable_table_admin_client_config.json

```diff
@@ -94,7 +94,7 @@
         },
         "GetIamPolicy": {
           "timeout_millis": 60000,
-          "retry_codes_name": "idempotent",
+          "retry_codes_name": "non_idempotent",
```

Comment:
this looks strange to me, please hold off on merging for now until I confirm the behavior here. Thanks!